code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -661,6 +661,14 @@
 	. = 1
 	..()
 
+/datum/reagent/toxin/capilletum
```

Comment:
```suggestion
/datum/reagent/toxin/capilletum
```
Document this for everyone. Just saying its a new chemical isn't enough of a description.
Like me saying "adds a new feature to the game"